docs/source/platform/schema-registry.mdx

```diff
@@ -44,6 +51,27 @@ id      schema        tag
 190330  example-4218  current
 ```
 
+## Registering a schema with Federation
+
+If you're running a distributed GraphQL infrastructure, where GraphQL microservices [compose](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/#fetch-service-capabilities) to form a complete schema, there's functionality to register each service's partial schema and still get all of the benefits of the schema registry.
+
+To push a single implementing service to Apollo, run `apollo service:push` with the `--serviceName` and `--endpoint` tags. The CLI will know where to fetch your local partial schema based on this endpoint. Note that running `apollo service:push` on any implementing service also registers the overall schema of the graph and writes files to GCS.
+
+As an example, running service push on a local “Accounts” implementing service might look like: 
+
+```
+~$apollo service:push --serviceName="Accounts" --endpoint="http://localhost:4001/"
+  ✔ Loading Apollo Project
+  ✔ Uploading service to Engine
+
+
+id      schema                        tag
+──────  ────────────────────────────  ───────
+190331  service-list-federation-demo  current
+```
+
+Pushing an implementing service to Apollo will update the [Services tab](https://www.apollographql.com/docs/platform/federation/#inspecting-your-graph/).
```

Comment:
Please triple check this link

Reply:
should be good 👍 	
----------
poking @trevorblades to look into allowing us to mirror the prod url structure so we can use relative links that are testable locally (if this were `/docs/platform/federation/#inspecting-your-graph` then we can know if it works or not when it's running on localhost)